examples/display-dsi-command-teartest-stm32h747i-disco.rs

```diff
@@ -0,0 +1,358 @@
+//! This example uses the embedded-graphics library to draw fast moving objects on
+//! an external display. The external display is connected through the DSI link.
+//! DSI Adapted command mode is used to refresh the display only when needed.
+//!
+//! When using the display in Portrait mode, no diagonal tearing line should be visible.
+//!
+//! While in Landscape mode diagonal line is visible, because display is still refreshed
+//! in 480px lines from display's own graphics RAM, while writes are now 800px high.
+//!
+//! Run command: cargo embed --release --features="stm32h747cm7,dsi,log,ltdc,fmc,rm0399,smps,example-smps,log-rtt,rt,rtc" --example display-dsi-command-teartest-stm32h747i-disco
```

Comment:
```suggestion
//! Run command: cargo embed --release --features="stm32h747cm7,dsi,log,ltdc,fmc,example-smps,log-rtt,rt,rtc" --example display-dsi-command-teartest-stm32h747i-disco
```

These two feature gates are enabled by `stm32h747cm7`, so it's not needed and could be confusing to include them here